src/main/java/pl/sda/travelagency/Departure.java

```diff
@@ -0,0 +1,32 @@
+package pl.sda.travelagency;
+
+import javax.persistence.Entity;
+
+//@Entity
+public class Departure {
+    private City city;
```

Comment:
To tez bym uprościł, w klasie Trip raczej bym dał pole DepartureCity - na chwilę obecną wystarczy